unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NFTPromptHUD/NFTPromptHUDView.cs

```diff
@@ -189,44 +190,57 @@ private void SetNFTInfo(NFTInfo info, string comment)
 
     private IEnumerator FetchNFTImage(NFTInfo nftInfo)
     {
+        Debug.Log("pravs - NFTPromptHUDView.FetchNFTImage() - 1 - Waiting fetch...");
```

Comment:
I guess these `Debug.Logs` should be removed, right?

Reply:
Done